src/components/ArchivePage/EventInfoItem.js

```diff
@@ -45,9 +47,11 @@ const useStyles = makeStyles(theme => ({
 
 function EventInfoItem({ name, mainLink, tags, director, workshops }) {
 	const classes = useStyles();
-	return <Accordion classes={{ root: classes.paperRoot }}>
+	const [isExpanded, setExpanded] = useState(true);
+	return <Accordion classes={{ root: classes.paperRoot }}
+				onChange={() => setExpanded(isExpanded => !isExpanded)}>
```

Comment:
nit:

```suggestion
	return <Accordion classes={{ root: classes.paperRoot }}
	                  onChange={() => setExpanded(isExpanded => !isExpanded)}>
```

Use spaces for indentations beyond the `return` keyword, as the size of tabs can change based on the editor:

![image](https://user-images.githubusercontent.com/1538624/104875263-dc04e300-5922-11eb-8d61-060d9cb48799.png)

Reply:
linter error :/ <br>
<img width="729" alt="Screen Shot 2021-01-17 at 9 26 16 PM" src="https://user-images.githubusercontent.com/40647794/104875679-b9ff6680-590a-11eb-9b94-36fc5f567633.png">

Reply:
Bah

Reply:
The original is fine then :)